apps/projects/app/components/Content/Issues.js

```diff
@@ -159,14 +159,16 @@ class Issues extends React.PureComponent {
       // if there are no Status filters, all issues pass
       if (Object.keys(filters.statuses).length === 0) return true
       // should bountyless issues pass?
-
       const status = bountyIssueObj[issue.number] ? bountyIssueObj[issue.number] : 'not-funded'
       if ('not-funded' in filters.statuses && !bountyIssueObj[issue.number])
         return true
       // if issues without a status should not pass, they are rejected below
       if (status === 'not-funded') return false
```

Comment:
This line isn't needed, it'll hit the return false regardless of this line being here.